runtime/calamari/src/impls.rs

```diff
@@ -31,11 +37,19 @@ impl OnUnbalanced<NegativeImbalance> for DealWithFees {
 	fn on_unbalanceds<B>(mut fees_then_tips: impl Iterator<Item = NegativeImbalance>) {
 		if let Some(fees) = fees_then_tips.next() {
 			// for fees, 40% to treasury, 60% to author
-			let mut split = fees.ration(40, 60);
```

Comment:
@stechu i know you had a suggestion to make these part of the `DealWithFee` struct, however there wouldn't be a way to access them, because I can't change the signature of `on_unbalanceds` trait function.

Reply:
yeah, makes sense. The function did pass self in.